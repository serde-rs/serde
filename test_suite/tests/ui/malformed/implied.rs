use serde_derive::Serialize;

#[derive(Serialize)]
#[serde(implied(unknown))]
struct S {
    x: (),
}

fn main() {}
