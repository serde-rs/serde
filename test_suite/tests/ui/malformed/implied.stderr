error: malformed implied attribute, expected `implied(key = ..., value = ...)`
 --> tests/ui/malformed/implied.rs:4:17
  |
4 | #[serde(implied(unknown))]
  |                 ^^^^^^^
